CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [3.3.0]
+
+**Improvements**
+Add cron scheduler to change the status of pending order to canceled
```

Comment:
If this is based on a configuration value, please state so.

Reply:
This has been updated